RadixWallet/Features/AssetTransferFeature/Components/TransferAccountList/ReceivingAccount/ReceivingAccount+Reducer.swift

```diff
@@ -76,65 +72,45 @@ public struct ReceivingAccount: Sendable, FeatureReducer {
 	}
 }
 
-extension ReceivingAccount.State.Account {
+extension AssetsTransfersRecipient {
```

Comment:
Suggest a better name please :) this is inside Sargon, so must not use `.` in the name. (Rust does not work with `.` type names.).

It is currently only used for transfers. So I think the name is pretty good. And we can change it if we were to use it in other contexts? 

But you might convince me if you suggest an awesome name :) - since we cannot use `.` in Rust, the name cannout be `Account` for that is `Profile.Netork.Account`'s name in Sargon.

Reply:
TransferAccount...

Reply:
I went with `AccountOrAddressOf`:

```rust
```

 - we can always create a typealias for that in the context of Transaction in iOS if we want.

https://github.com/radixdlt/sargon/commit/d2c0166c0a5c499d634ef83d331315ef570694d9